### DISCUSSION TITLE
Which Version for Ender 3 V2 with Creality CRTouch?

### DISCUSSION THREAD

**Discussion by ChrisThomas78** [Jyers/Marlin#1271] [Q&A] [answered]
Hi there,
I´ve recently purchased an Ender 3 V2 mit CRTouch (NOT BLTouch!)
Which Version should I download and install. Board Version is not yet known...Printer is being delivered the next few days...

Thx in advance.

Regards
Chris

**Comment by adam3654** [comment, marked as answer]
There is no difference functionally between the two. Just use your desired bltouch bin. I swapped mine out using the old bltouch cable, and just had to adjust the z offset after.

**Reply by ChrisThomas78** [reply]
Thank you adam for that fast response. So I just check which Board I will have and Download the bin and update the firmware... Adjust all offsets and I will be ready to go....

**Reply by jiriteach** [reply]
Does the probe offset have to be changed? If so - whats the probe offset of the CR Touch? Considering buying one.
